src/qibolab/__init__.py

```diff
@@ -39,6 +39,10 @@ def create_platform(name, runcard=None):
     if name == "dummy":
         from qibolab.dummy import create_dummy
 
+        return create_dummy()
+    if name == "dummy2":
```

Comment:
I would suggest a more meaningful name compared to `dummy2`.
Maybe `dummy_couplers` ?